### ISSUE TITLE
Edit form modal jumps to the top of the RTE when text pasted in the end and also on FF by just typing 

### ISSUE THREAD

**Issue by asotog** [craftercms/craftercms#327]
STEPS: 
1. Go to any content page with RTE and try to edit the content on the RTE field
2. Insert a paragraph on the bottom and apply some font style on it 
3. Select all the paragraph and right click on it (later select the copy option) 
4. Paste the just copied paragraph at the bottom using the keyboard shortcut (command + v ) 
5. Notice that the application jumps up to the top of the RTE

**Comment by asotog** [comment]
This is issue is happening on Firefox, latest version, but when you have long rte and start typing at the bottom

**Comment by asotog** [comment]
Fixed critical issue on FF, where any text typed was scrolling/jumping up the window

**Comment by asotog** [comment]
When removing the `paste` plugin from the plugin list, the other pasting issue disappears, so this seems related to tinymce paste plugin:
ln 458

```
var pluginList = "paste, noneditable, cs_table, cs_inlinepopups, lists, ";
...
```

**Comment by asotog** [comment]
Tested again, still see the issue using clean environment

```
#Generated by Maven
#Wed Dec 06 21:14:27 EST 2017
version=3.0.3
groupId=org.craftercms
artifactId=crafter-studio
```

**Comment by sumerjabri** [comment]
@asotog Kindly use 3.0.4 https://craftersoftware.com/downloads

**Comment by asotog** [comment]
thanks will take a look

**Comment by asotog** [comment]
after checking 3.0.4 behaviour persists there as well :(

**Comment by sumerjabri** [comment]
@asotog Scheduled for the next release. In the future, please follow the issue template provided during issue creation for consistency and legibility and to avoid too much back and forth. Stay tuned.

**Comment by jvega-crafter** [comment]
Checked and fix is in 3.0.4, confirmed with @asotog, proceeding to close issue

**Comment by asotog** [comment]
thanks so much
